content/configuration/auth-sso.md

```diff
@@ -80,7 +80,7 @@ OpenID Connect (OIDC) is an authentication protocol built on OAuth 2.0, and shou
 | `AUTH_<PROVIDER>_CLIENT_ID`                 | Client identifier for the external service.                                                                                                                                                                               |                        |
 | `AUTH_<PROVIDER>_CLIENT_SECRET`             | Client secret for the external service.                                                                                                                                                                                   |                        |
 | `AUTH_<PROVIDER>_CLIENT_*`                  | Client options overrides passed to the [underlying client](https://github.com/panva/openid-client).                                                                                                                       |                        |
-| `AUTH_<PROVIDER>_PRIVATE_KEYS`              | An array of JSON Web Key Set (JWKS) private keys used to sign client assertions when `AUTH_<PROVIDER>_CLIENT_TOKEN_ENDPOINT_AUTH_METHOD` is set to `private_key_jwt`.                                                     |                        |
+| `AUTH_<PROVIDER>_PRIVATE_KEYS`              | An array of JSON Web Key Set (JWKS) private keys used to sign client assertions when `AUTH_<PROVIDER>_CLIENT_TOKEN_ENDPOINT_AUTH_METHOD` is set to `private_key_jwt`. The env needs to be prefixed with `json:`.          |                        |
```

Comment:
@licitdev Could we add this env var to the casting list in `@directus/env` so it's cast as `json` by default? https://github.com/directus/directus/blob/52ed9656f59b1b392b5969114fffce52c89281b6/packages/env/src/constants/type-map.ts#L7